src/tests/components/Auth/FormInputGroup/index.test.js

```diff
@@ -52,17 +55,19 @@ describe('When TextInput component is mounted', () => {
   it('renders forgot password link if inputType is password', () => {
     const passInputType = 'password';
     const { getByText } = render(
-      <Formik initialValues={ initialValues } onSubmit={ mockSubmit }>
-        <Form>
-          <FormInputGroup
-            label={ label }
-            inputId={ inputId }
-            inputType={ passInputType }
-            name={ name }
-            forgotPassLink={ true }
-          />
-        </Form>
-      </Formik>
+      <Router history={ history }>
```

Comment:
This line was needed to add to make the tests pass as `NavLink` is being used in `FormInputGroup/index.js` to visit the `ForgotPassword` page.